akka-actor/src/main/mima-filters/2.5.14.backwards.excludes

```diff
@@ -1,3 +1,6 @@
+#25435 - Adding maximum restart attempts to BackoffSupervisor
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.pattern.BackoffOptions.withMaxNrOfRetries")
```

Comment:
I wasn't sure if it's safe to exclude this binary compatibility issue. Could you take a look?
@raboof @ktoso